### PR TITLE
use strong params in product_scopes_controller, clean views as per spree 2.x

### PR DESCRIPTION
**Pull request by priyank-gupta**
Using strong params in product_scopes_controller (add private method product_scope_params).
Using link_to delete and other view changes as in spree 2.x

### CODE REVIEW

File: app/controllers/spree/admin/product_scopes_controller.rb [hunk 1]
```diff
@@ -34,6 +34,12 @@ def destroy
         end
       end
 
+      private
+
+      def product_scope_params
+        params.require(:product_scope).permit(:name)
```
Comment (radar) [inline app/controllers/spree/admin/product_scopes_controller.rb]: If this is an admin controller I think we can just go ahead and `params.permit!`. We only care about restricting parameters tightly on the frontend. We trust admins to behave :)

**Comment by radar** [comment]
Thanks @priyank-gupta. This patch looks good to me. Please just fix up that one thing and then I will go ahead and merge it in.

**Comment by priyank-gupta** [comment]
@radar, permitting all params for product scope in product_scopes_controller.

**Comment by radar** [comment]
Thanks @priyank-gupta. Added this to 2-1-stable and master now.

**Pull request closed without merging**